diff -up locale/locale.pri.orig locale/locale.pri
--- locale/locale.pri.orig	2012-01-06 09:44:16.966957462 +0200
+++ locale/locale.pri	2012-01-06 10:33:17.215239541 +0200
@@ -4,7 +4,7 @@ INCLUDEPATH += $$PWD
 DEPENDPATH += $$PWD
 
 # ls -1 *.ts | tr '\n' ' '
-TRANSLATIONS += ca.ts ca_ES.ts da.ts de_DE.ts el.ts en_US.ts es.ts es_AR.ts es_ES.ts fi_FI.ts fr.ts gl.ts he_IL.ts hr.ts hu.ts ia.ts id.ts id_ID.ts it.ts jv.ts ka_GE.ts nb.ts nl.ts pl.ts pl_PL.ts pt.ts pt_BR.ts ro.ts ru.ts sl.ts sq.ts sr.ts sv_SE.ts te.ts tr.ts uk_UA.ts zh_CN.ts 
+TRANSLATIONS += ca.ts ca.ts da.ts de.ts el.ts en.ts es.ts fi.ts fr.ts gl.ts he.ts hr.ts hu.ts ia.ts id.ts it.ts jv.ts ka.ts nb.ts nl.ts pl.ts pt.ts ro.ts ru.ts sl.ts sq.ts sr.ts sv.ts te.ts tr.ts uk.ts zh_CN.ts
 isEmpty(QMAKE_LRELEASE) {
     win32:QMAKE_LRELEASE = $$[QT_INSTALL_BINS]\lrelease.exe
     else:QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
