diff -up minitube/minitube.pro.orig minitube/minitube.pro
--- minitube/minitube.pro.orig	2013-02-06 17:15:37.339315558 +0200
+++ minitube/minitube.pro	2013-02-06 17:28:15.052786122 +0200
@@ -15,7 +15,8 @@ DEFINES += QT_STRICT_ITERATORS
 
 TARGET = minitube
 QT += network xml phonon
-include(src/qtsingleapplication/qtsingleapplication.pri)
+# include(src/qtsingleapplication/qtsingleapplication.pri)
+CONFIG += qtsingleapplication
 HEADERS += \
     src/video.h \
     src/searchlineedit.h \
diff -up minitube/src/mainwindow.cpp.orig minitube/src/mainwindow.cpp
--- minitube/src/mainwindow.cpp.orig	2013-02-06 17:18:03.905070590 +0200
+++ minitube/src/mainwindow.cpp	2013-02-06 17:19:14.886417012 +0200
@@ -146,7 +146,7 @@ MainWindow::MainWindow() :
     mouseTimer->setSingleShot(true);
     connect(mouseTimer, SIGNAL(timeout()), SLOT(hideMouse()));
 
-    QTimer::singleShot(0, this, SLOT(checkForUpdate()));
+    //QTimer::singleShot(0, this, SLOT(checkForUpdate()));
 
 }
 
