diff -up minitube/locale/locale.pri.orig minitube/locale/locale.pri
--- minitube/locale/locale.pri.orig	2010-09-26 10:51:58.346654903 +0300
+++ minitube/locale/locale.pri	2010-09-26 11:25:00.978607272 +0300
@@ -14,7 +14,7 @@ TRANSLATIONS += \
     cs_CZ.ts \
     uk.ts \
     he_IL.ts \
-    lat.ts \
+    lv.ts \
     hr_HR.ts \
     es.ts \
     gl.ts \
@@ -37,8 +37,8 @@ isEmpty(QMAKE_LRELEASE) {
 }
 
 updateqm.input = TRANSLATIONS
-updateqm.output = build/target/locale/${QMAKE_FILE_BASE}.qm
-updateqm.commands = $$QMAKE_LRELEASE ${QMAKE_FILE_IN} -qm build/target/locale/${QMAKE_FILE_BASE}.qm
+updateqm.output = build/target/locale/minitube_${QMAKE_FILE_BASE}.qm
+updateqm.commands = $$QMAKE_LRELEASE ${QMAKE_FILE_IN} -qm build/target/locale/minitube_${QMAKE_FILE_BASE}.qm
 updateqm.CONFIG += no_link target_predeps
 QMAKE_EXTRA_COMPILERS += updateqm
 
