diff -up minitube/locale/locale.pri.orig minitube/locale/locale.pri
--- minitube/locale/locale.pri.orig	2011-03-19 13:37:43.035724368 +0200
+++ minitube/locale/locale.pri	2011-03-19 13:39:05.132716910 +0200
@@ -12,8 +12,8 @@ isEmpty(QMAKE_LRELEASE) {
 }
 
 updateqm.input = TRANSLATIONS
-updateqm.output = build/target/locale/${QMAKE_FILE_BASE}.qm
-updateqm.commands = $$QMAKE_LRELEASE ${QMAKE_FILE_IN} -qm build/target/locale/${QMAKE_FILE_BASE}.qm
+updateqm.output = build/target/locale/minitube_${QMAKE_FILE_BASE}.qm
+updateqm.commands = $$QMAKE_LRELEASE ${QMAKE_FILE_IN} -qm build/target/locale/minitube_${QMAKE_FILE_BASE}.qm
 updateqm.CONFIG += no_link target_predeps
 QMAKE_EXTRA_COMPILERS += updateqm
 
